coco/scheduler.py

```diff
@@ -63,10 +64,12 @@ def _gen_timers(self, endpoints):
 
         for edpt in endpoints.values():
             if edpt.schedule is not None:
+                # Check for values
                 if edpt.values is not None:
                     raise ValueError(
```

Comment:
Should we exit here instead of raising an Exception?

Reply:
sure, I can change that.